### ISSUE TITLE
ICD-10 initial and subsequent encounters (especially for self-harm) have the same mappings

### ISSUE THREAD

**Issue by ChristopheLambert** [OHDSI/Vocabulary-v5.0#881]
**Describe the problem in content or desired feature:**
The current ICD10CM to SNOMED mappings lose granularity between initial encounter, subsequent encounter, and sequelae. Specifically, for the use case of self-harm outcomes, initial and subsequent encounters in ICD10CM map to the same SNOMED codes, making it appear as if there's a new self-harm event when it's merely a follow-up. This can create biases in studies such as inferring that self-harm risk is much higher than it truly is for those who have past self-harm -- confusing subsequent visits with additional events.

**How to find it:**
The issue can be observed by contrasting concept_id 45600200  [X71.3XXA - Intentional self-harm by drowning and submersion in natural water, initial encounter](https://atlas-demo.ohdsi.org/#/concept/45600200) and concept_id 45566263 [X71.3XXD - Intentional self-harm by drowning and submersion in natural water, subsequent encounter](https://atlas-demo.ohdsi.org/#/concept/45566263). They both map to concept_id 4303690 [Intentionally harming self](https://atlas-demo.ohdsi.org/#/concept/4303690).

**Expected adjustments:**
Recommend that codes that relate to subsequent encounters (in their one-to-many mapping) map to the SNOMED OMOP concept_id 35626371 - https://atlas-demo.ohdsi.org/#/concept/35626371, "Subsequent encounter by subject", or a suitable alternate that captures this. This would be consistent with the practice of having sequelae map to:  https://atlas-demo.ohdsi.org/#/concept/435119  "Late effect of injury" or a more specific term (when available), for example, https://atlas-demo.ohdsi.org/#/concept/435446 "Late effect of self-inflicted injury", which should allow one to differentiate these events from the initial injury. This will affect hundreds of mappings. For example, it is reported this would also be helpful for other conditions, such as myocardial infarction, where one would not want to confuse follow-up visits with additional myocardial infarctions.

**Screenshots:** 
Not applicable.

**Additional context:**
It has been correctly noted by @hripcsa  that many health systems will not capture this nuance of initial versus subsequent encounters correctly in the source coding. However, to the extent those terms are used by some health systems in the source coding, machine learning models might use these positive examples in machine learning algorithms to detect and correct this mislabelling if we do not lose this information in the mapping, see for example https://www.ohdsi.org/2022showcase-77/. All of our research on self-harm and other mental health conditions has avoided using the OMOP standard vocabularies because of this type of loss of granularity. However, using source_concept_id values is problematic for international OHDSI network studies.

**Comment by cgreich** [comment]
@ChristopheLambert: You are right, ICD-10-CM the subsequent encounters are defined as “encounters after the patient has received active treatment of the injury and is receiving routine care for the injury during the healing or recovery phase". There is no new onset or event. And you would want to not confuse the two.

However, the baby is probably out with the bath water. Diagnoses are routinely repeated. In chronic diseases, it doesn't matter, the diabetes today is the same as the one a week ago. Acute conditions, like self-harm or asthma, don't work that way. To distinguish one onset from the other you have to apply some phenotyping gymnastics (e.g. setting an expiration time). 

I would strongly vote against having second encounters being their own diagnoses, they are not. The only thing we could think of is mapping them to 0 (not mapping them). The problem with that one is that you need to make sure you have a first encounter, otherwise you'd undercount. 

Not easy.

**Comment by ChristopheLambert** [comment]
@cgreich can you clarify what you are strongly voting against? My original comment acknowledged that gymnastics can be used to distinguish initial versus subsequent encounters. ICD-10 distinguishes between initial, subsequent, and sequelae -- OMOP vocabulary mappings currently distinguish sequale, but blend together initial and subsequent encounters. And these are done by one-to-many mappings. Why not just tack on the https://atlas-demo.ohdsi.org/#/concept/35626371 "Subsequent encounter by subject" term as an additional one-to-many mapping to what we already have for the subsequent encounters?

**Comment by cgreich** [comment]
@ChristopheLambert:

I have full empathy for what you want, but I am voting against overloading the meaning of a record in CONDITION_OCCURRENCE and against post-coordination with no mechanism to actually post-coordinate effectively.

- A Condition record does not imply that there is a new onset of something. For chronic diseases, you will have many many records of, say, diabetes, and obviously it is still the same diabetes. To distinguish, you need to use phenotyping algorithms.
- You will say "but some codes tell me that there is no new onset, but the old one is still ongoing". I agree, another "this is not an onset" record like concept 35626371 could capture that fact. Except it would stand alone, with no connection to the record it refers to. You'd have to give it a FACT_RELATIONSHIP. Have you ever considered using one of those in an analytic?
- Sequelae are an entirely different situation. These are true new Conditions distinct from the initial one. The reason we map one-to-many is that the primary diagnosis still stands like in the subsequent encounter situation, but a new fresh onset of something else happened.

Should we consider a new Condition Status "Existing ongoing diagnosis"?

**Comment by ChristopheLambert** [comment]
@cgreich Thank you very much for the ongoing dialog and empathy!

I agree a condition record does not imply there is a new onset for all conditions. However, the diabetes example you provided pertains to a chronic disease. Conditions such as a suicide attempt or a myocardial infarction, unlike chronic diseases, are transient events and do not persist. How one analyzes and studies transient events like these versus chronic conditions can have important differences. For instance, researchers are very interested in assessing the recurrence of additional suicide attempts and myocardial infarctions through techniques such as survival analysis or Poisson regression, but you would not analyze diabetes or other chronic conditions using these same techniques, as concepts like 'time to next diabetes diagnosis' are not applicable.

Numerous self-harm/suicidality condition mappings are currently using one-to-many relationships. For example, specific substance intentional poisonings in ICD-10 that have no SNOMED counterparts have the specific SNOMED poisoning concept (without intent) plus the concept for intentionally harming self. I do not understand what you said about overloading the meaning of a record -- it seems like that is already being done. One has to look at both terms to understand what is going on (e.g. specific poison substance + intent), and I was suggesting doing the same thing with capturing a follow-up visit. This seemed like a way to not disrupt the existing mappings while adding an additional one-to-many mapping to capture that it is a follow-up visit and not a new event. People who do not care about that fact can continue to analyze their data as before.

I have no familiarity with using the FACT_RELATIONSHIP -- would it be some custom thing that would not port over the OHDSI network? I am not attached to the specific remedy I suggested, but would like a remedy. The loss of information in the mappings makes any use of logic to distinguish initial events versus subsequent encounters impossible without using source codes or making assumptions that people do not have recurrences in some arbitrary time window which does not make sense. For example, you could have a second myocardial infarction the next day or 2 years later.

If you were studying these questions and not chronic diseases, how would you recommend adjusting the vocabulary in a portable way (for network studies) to not lose this information?

Thanks!

**Comment by cgreich** [comment]
@ChristopheLambert:

> Conditions such as a suicide attempt or a myocardial infarction, unlike chronic diseases, are transient events and do not persist. 

Absolutely. And the phenotype definitions go to great length to figure out the cohort end date. Because usually you don't get that "subsequent" encounter, and still have to figure out if a repeat condition concept is still the same MI, or a new one. Folks use expected timing for a condition (one or two weeks for MI), or they look for the gap between condition records. Or they use some other data such as procedures and lab tests to figure out when the MI ended.

> it seems like that is already being done. One has to look at both terms to understand what is going on (e.g. specific poison substance + intent

Can you explain what is going on there?

The problem I was alluding to was if you have a separate record "Not new onset/subsequent encounter" it is not clear which condition it belongs you on that day. With poison and suicide that may be more obvious. Do I get that right?

> Should we consider a new Condition Status "Existing ongoing diagnosis"?

That was my suggestion. ATLAS would be able to figure that out.

**Comment by ChristopheLambert** [comment]
To explain what I mean by multiple terms being needed to determine the condition, consider how the vocabulary currently deals with these three ICD10CM codes:

T58.02XA "Toxic effect of carbon monoxide from motor vehicle exhaust, intentional self-harm, **initial encounter**", concept_id 45565724, maps to 3 different standard SNOMED standard codes in the concept_relationship table:
- 439235 Self inflicted injury
- 4152376 Intentional self poisoning
- 4320826 Carbon monoxide poisoning from motor vehicle exhaust

ICD10 T58.02XD "Toxic effect of carbon monoxide from motor vehicle exhaust, intentional self-harm, **subsequent encounter**" concept_id 45604429 maps to the same 3 codes:
- 439235 Self inflicted injury
- 4152376 Intentional self poisoning
- 4320826 Carbon monoxide poisoning from motor vehicle exhaust

This loses the fact that it is a subsequent encounter, mapping to the very same terms as the initial encounter.

ICD10CM T58.02XS "Toxic effect of carbon monoxide from motor vehicle exhaust, intentional self-harm, **sequela**", concept_id 45604430 maps to:
- 435446 Late effect of self inflicted injury
- 4180771 Disorder due to and following poisoning

This third pair of mappings captures both poisoning and that it was intentional (though loses the carbon monoxide aspect), but has a SNOMED concept for sequela of self inflicted injury.

All I am suggesting is that you update the subsequent encounter mappings to add one additional mapping, for example:

ICD10 T58.02XD "Toxic effect of carbon monoxide from motor vehicle exhaust, intentional self-harm, **subsequent encounter**" concept_id 45604429, becomes:
- 439235 Self inflicted injury
- 4152376 Intentional self poisoning
- 4320826 Carbon monoxide poisoning from motor vehicle exhaust
- **35626371 Subsequent encounter by subject**

Unfortunately, 35626371 is an Observation, However, I checked and **it is actually ALREADY being used this way**, but just for two ICD10CM codes:
- S20.374D Other superficial bite of middle front wall of thorax, subsequent encounter, concept_id 725532
- T40.493D Poisoning by other synthetic narcotics, assault, subsequent encounter, concept_id 725597
Both of these have maps_to relationships to 35626371 "Subsequent encounter by subject".

**So OMOP is already doing what I suggested, just not consistently!**  May I suggest that 35626371 "Subsequent encounter by subject" be moved to the Condition domain and that it be used for all subsequent encounter codes in ICD9/10? This uses an existing standard SNOMED code already being used for my use case and one does not need to create some new term non-existent in SNOMED such as "Existing ongoing diagnosis".
